### PR TITLE
create react app

### PR DESCRIPTION
**Pull request by vovkis13**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+
+# dependencies
+/node_modules
+/.pnp
+.pnp.js
```
Comment (kerrimov) [inline .gitignore:6]: Do we really need these files?

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+# production
+/build
+
+# misc
+.DS_Store
+.env.local
+.env.development.local
+.env.test.local
+.env.production.local
```
Comment (kerrimov) [inline .gitignore:19]: For what purposes we need these files here?

File: README.md [hunk 3]
```diff
@@ -1 +1,46 @@
-# marvel-heroes
```
Comment (kerrimov) [inline README.md:1]: Please change `README.md` file according to your project.

File: package.json [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+      "not dead",
+      "not op_mini all"
+    ],
+    "development": [
+      "last 1 chrome version",
+      "last 1 firefox version",
+      "last 1 safari version"
+    ]
+  }
+}
```
Comment (kerrimov) [inline package.json:43]: No need for this items

File: public/manifest.json [hunk 5]
```diff
@@ -0,0 +1,25 @@
+{
```
Comment (kerrimov) [inline public/manifest.json:1]: This file can be deleted

File: public/robots.txt [hunk 6]
```diff
@@ -0,0 +1,3 @@
+# https://www.robotstxt.org/robotstxt.html
```
Comment (kerrimov) [inline public/robots.txt:1]: This file can be deleted

File: public/index.html [hunk 7]
```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
```
Comment (kerrimov) [inline public/index.html:3]: This file can be deleted

File: src/reportWebVitals.ts [hunk 8]
```diff
@@ -0,0 +1,15 @@
+import { ReportHandler } from 'web-vitals';
+
+const reportWebVitals = (onPerfEntry?: ReportHandler) => {
```
Comment (kerrimov) [inline src/reportWebVitals.ts:3]: can be deleted

File: src/App.tsx [hunk 9]
```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import logo from './logo.svg';
+import './App.css';
```
Comment (kerrimov) [inline src/App.tsx:3]: this file can be renamed or deleted

File: src/App.css [hunk 10]
```diff
@@ -0,0 +1,38 @@
+.App {
```
Comment (kerrimov) [inline src/App.css:1]: this file can be deleted

File: src/App.test.tsx [hunk 11]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import App from './App';
+
+test('renders learn react link', () => {
```
Comment (kerrimov) [inline src/App.test.tsx:5]: can be deleted

**Comment by kerrimov** [comment]
`package-lock` should be deleted as well

File: .gitignore (hunk 1 above)
Comment (vovkis13) [reply .gitignore:6]: No, we don't use any libraries to develop SharePoint Framework web parts

File: .gitignore (hunk 2 above)
Comment (vovkis13) [reply .gitignore:19]: DS_Store contains settings of MacOS environment, .env* files can contain data like database credentials and API keys that needs to be secure

File: .gitignore [hunk 12]
```diff
@@ -0,0 +1,24 @@
...
+
+npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
+
+package-lock.json
```
Comment (kerrimov) [inline .gitignore:23]: Your project setupped by `yarn`, so no need to include `package-lock.json` here.

File: .gitignore [hunk 13]
```diff
@@ -0,0 +1,24 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
```
Comment (kerrimov) [inline .gitignore:1]: this line can be removed

File: README.md [hunk 14]
```diff
@@ -1 +1,5 @@
...
\ No newline at end of file
+Marvel-Heroes
+
+- It's a react web application allowing users to look for marvel superheroes in
+  order to see comic books where the hero appeared
+- Uses Marvel API https://developer.marvel.com/docs
```
Comment (kerrimov) [inline README.md:5]: How can I run a project? Please include this info to the `README`

File: src/index.tsx [hunk 15]
```diff
@@ -0,0 +1,17 @@
...
+  <React.StrictMode>
+    <App />
+  </React.StrictMode>,
+  document.getElementById('root')
+);
+
+// If you want to start measuring performance in your app, pass a function
+// to log results (for example: reportWebVitals(console.log))
+// or send to an analytics endpoint. Learn more: https://bit.ly/CRA-vitals
+reportWebVitals();
```
Comment (kerrimov) [inline src/index.tsx:17]: this part of the code can be removed for now

**Review by kerrimov — APPROVED** [review]
For now, return `index.html`

File: public/index.html (hunk 7 above)
Comment (kerrimov) [reply public/index.html]: Save for now

File: .gitignore (hunk 12 above)
Comment (vovkis13) [reply .gitignore:23]: package-lock.json deleted

File: .gitignore (hunk 13 above)
Comment (vovkis13) [reply .gitignore:1]: removed

**Pull request merged**